docker-compose.override.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
```

Comment:
isn't this always applied if existing? So would also be read in `ci`?

Reply:
Not when using `-f`.

Reply:
Reasoning on how the 3 files are intended to match an environment:
- `dev`: `docker-compose.yml`, `docker-compose.override.yml`
- `ci`: `docker-compose.yml`, `docker-compose.ci.yml`
- demoing the application: just `docker-compose.yml`